src/components/editor/MonacoEditor.vue

```diff
@@ -69,15 +70,38 @@ async function getFileContent() {
     .then((fileInput) => fileInput.content);
 }
 
+/**
+ * Initialize Monaco languages from plugins.
+ */
+function initLanguages() {
+  getPlugins()
+    .filter((plugin) => plugin.configuration.editor.syntax !== null)
+    .forEach((plugin) => {
+      const language = plugin.configuration.editor.syntax.name;
+      monaco.languages.register(plugin.configuration.editor.syntax.languageSettings);
+      monaco.languages.setLanguageConfiguration(
+        language,
+        plugin.configuration.editor.syntax.languageConfiguration,
+      );
+      monaco.languages.setMonarchTokensProvider(
+        language,
+        plugin.configuration.editor.syntax.tokenProvider,
+      );
+    });
+}
+
 /**
  * Setup monaco editor.
+ * Register terraform language syntax colorizer
  * @return {Promise<void>} Promise with nothing on success otherwise an error.
  */
 async function createEditor() {
+  initLanguages();
```

Comment:
weird at the beginning of the function. Better be done just before create(). OR EVEN done outside of the function, before createEditor().

Reply:
done